go.mod

```diff
@@ -8,7 +8,7 @@ require (
 	github.com/google/go-querystring v0.0.0-20170111101155-53e6ce116135 // indirect
 	github.com/pkg/errors v0.9.1
 	github.com/urfave/cli/v2 v2.3.0
-	golang.org/x/mod v0.2.0
+	golang.org/x/mod v0.6.0-dev.0.20220106191415-9b9b3d81d5e3
```

Comment:
any reason not to use the latest stable which is v0.5.1 https://pkg.go.dev/golang.org/x/mod ?

Reply:
It was bumped automatically as a dependency of tools. https://github.com/golang/tools/blob/v0.1.10/go.mod#L7

tools 0.1.9 uses mod 0.5.1. I tested that with my project and it still works. I can downgrade it if you want to use the stable version.

Reply:
I see. Will merge then, thanks!